sympy/core/power.py

```diff
@@ -513,6 +513,11 @@ def _eval_is_extended_real(self):
                 return self.exp.is_imaginary
             return
         real_e = self.exp.is_extended_real
+        if real_b == False and real_e:
+            if self.exp == S.Half:
+                return False
+            if type(self.exp) == Rational and self.exp.numerator == 1:
+                return False
```

Comment:
There is already a `if real_b is False and real_e` case below so this should be moved there.

These two cases can be combined with `if isinstance(self.exp, Rational) and self.exp.p == 1`.

Perhaps the code there using `arg` should be removed in favour of this code checking for Rational and numerator 1.

Reply:
Hi, thanks for the feedback - i somehow missed that case, sorry.

Regarding the part with `arg`, i would be careful to remove it. As far as i can tell, it is relevant for computing the following:

```{python}
>>> expr = Pow(4*I+4, 8)
>>> print(expr.is_extended_real)
True
```

because when i remove it in favour of my code:
```{python}
>>> expr = Pow(4*I+4, 8)
>>> print(expr.is_extended_real)
None
```

I would thus suggest to not remove it

Reply:
It is expected that the assumptions system cannot resolve all queries and will sometimes return None. The problem with using `arg` here is that it creates a new expression which is not something that should happen within the assumptions system.